.build/tasks/generateHelp.PlatyPS.build.ps1

```diff
@@ -41,26 +48,162 @@ Task UpdateHelp {
+    }
+    else
     {
-        $BuildOutput = Join-Path -Path $ProjectPath.FullName -ChildPath $BuildOutput
+        # VersionedOutputDirectory may be [bool]'' but we can't tell where it's
+        # coming from, so assume the build info (Build.yaml) is right
+        $VersionedOutputDirectory = $BuildInfo['VersionedOutputDirectory']
+    }
+
+    $GetBuiltModuleManifestParams = @{
+        OutputDirectory          = $OutputDirectory
+        BuiltModuleSubDirectory  = $BuiltModuleSubDirectory
+        ModuleName               = $ProjectName
+        VersionedOutputDirectory = $VersionedOutputDirectory
+        ErrorAction              = 'Stop'
     }
 
-    if (-not (Split-Path -Path $HelpFolder -IsAbsolute))
+    $builtModuleBase = Get-SamplerBuiltModuleBase @GetBuiltModuleManifestParams
+    "`tBuilt Module Base        = '$builtModuleBase'"
+
+    $builtModuleManifest = Get-SamplerBuiltModuleManifest @GetBuiltModuleManifestParams
+    "`tBuilt Module Manifest    = '$builtModuleManifest'"
+
+    if ($builtModuleRootScriptPath = Get-SamplerModuleRootPath -ModuleManifestPath $builtModuleManifest)
     {
-        $HelpFolder = Join-Path -Path $SourcePath -ChildPath $HelpFolder
+        $builtModuleRootScriptPath = (Get-Item -Path $builtModuleRootScriptPath -ErrorAction SilentlyContinue).FullName
     }
 
-    Import-Module -Name $ProjectName -Force
-    Update-MarkdownHelpModule -Path $HelpFolder
+    "`tBuilt ModuleRoot script  = '$builtModuleRootScriptPath'"
+
+    $ModuleVersion = Get-BuiltModuleVersion @GetBuiltModuleManifestParams
+    $ModuleVersionObject = Split-ModuleVersion -ModuleVersion $ModuleVersion
+    $ModuleVersionFolder = $ModuleVersionObject.Version
+    $preReleaseTag       = $ModuleVersionObject.PreReleaseString
+
+    "`tModule Version           = '$ModuleVersion'"
+    "`tModule Version Folder    = '$ModuleVersionFolder'"
+    "`tPre-release Tag          = '$preReleaseTag'"
+
+    $HelpFolder = Get-SamplerAbsolutePath -Path $HelpFolder -RelativeTo $SourcePath
+    "`tHelp Folder Path         = '$HelpFolder'"
+
+    $HelpOutputFolder =  Get-SamplerAbsolutePath -Path $HelpOutputFolder -RelativeTo $OutputDirectory
+    "`tHelp output Folder        = '$HelpOutputFolder'"
+
+    $HelpOutputVersionFolder = Get-SamplerAbsolutePath -Path $ModuleVersion -RelativeTo $HelpOutputFolder
+    "`tHelp output Version Folder= '$HelpOutputVersionFolder'"
+
+    $HelpOutputCultureFolder = Get-SamplerAbsolutePath -Path $HelpCultureInfo -RelativeTo $HelpOutputVersionFolder
+    "`tHelp output Culture path  = '$HelpOutputCultureFolder'"
+
+    $generateHelpCommands = @(
+        '$env:PSModulePath = "{0}"' -f $Env:PSModulePath
+        'Import-Module -Name "{0}" -Force' -f $ProjectName
+        '$newMarkdownHelpParams = @{{ Module = "{0}"; OutputFolder = "{1}"; AlphabeticParamsOrder = ${2}; WithModulePage = ${3}; ExcludeDontShow = ${4}; Force = $true}}' -f $ProjectName, $HelpOutputCultureFolder, $true, $true, $true
```

Comment:
Maybe make `AlphabeticParamsOrder` a variable with a default value that can be overridden. I usually set that to `$false`.

Reply:
true, I just did not bother with the details for now :)

Reply:
still on the TODO list.